seqr_scripts/make_remap_file.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import csv
+import logging
+from os import replace
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("Make remap file")
+logger.setLevel(logging.INFO)
+
+
+def read_file(file_path: str):
+    """
+    Read and parse a two line text file containing:
+        - vcf ids separated by a commma and a space (", ") on the first line
+        - corresponding seqr ids separated by a comma and a space (", ") on the second line
+
+    This format for the text file was considered given how vcf ids and seqr ids are provided in airtable; it's easiest to copy each group into a single line, instead of a column for each group. Makes formatting seqr remap files much easier.
```

Comment:
how soon is Avery's work going into production? asking because of an edge case -- what if the IDs were accidentally entered out of order (or some other kind of human error?)
```
VCFIDS    SeqrIDs
vcf_sample_1, vcf_sample_2, vcf_sample_3    vcf_sample_1, vcf_sample_3, vcf_sample_2
```
this wouldn't get caught below (line 27 comparing the total number of vcf/seqr ids), and vcf_sample_2 would become vcf_sample_3, right?